aem-mock/core/src/test/java/io/wcm/testing/mock/aem/MockContentPolicyTest.java

```diff
@@ -19,8 +19,10 @@
  */
```

Comment:
It would be nice to also create a unit test, that tests the adaption of `ContentPolicy` to a custom model (since that's what you're fixing here).

Reply:
True, added a simple test with an example model